### DISCUSSION TITLE
New Libre 2 EU are not supported?

### DISCUSSION THREAD

**Discussion by psonnera** [NightscoutFoundation/xDrip#2840] [Q&A]
![image](https://github.com/NightscoutFoundation/xDrip/assets/33809361/41c3cf65-fb21-4c18-b6b1-003badfc233b)
Several reports (Portugal) of L2 sensor not useable with OOP2 (latest) and xDrip+ (latest nightly).

**Comment by parapenT1sta** [comment]
https://github.com/JohanDegraeve/xdripswift/pull/438/commits/a33b2c052216d6aa3d90c4ab3fd81c774e0a28e0

**Comment by tjcbento** [comment]
Same issue here (Portugal as well). Any idea if it is possible to workaround this?

**Reply by jaky1977** [reply]
O erro foi reportado de Portugal se vir a publicação, estamos á espera da possível resolução!

**Reply by parapenT1sta** [reply]
Como partilhei em cima a solução já existe no xDrip4iOS. E o @tzachi-dar já vai verificar no xDrip+/OOP2

![image](https://github.com/NightscoutFoundation/xDrip/assets/608650/f2c323a8-006d-47bc-9477-de27adf8e986)

**Comment by tzachi-dar** [comment]
Reference to the iphone solution :https://github.com/JohanDegraeve/xdripswift/pull/438/files

**Comment by tzachi-dar** [comment]
It seems that the iphone solution is based on recognizing all types based on the first digit of the serial number. On android we usually do it with 3 numbers.
So I need more logging to do it on the android way.

I need someone to go to setting->more settings ->extra tags for logging and add there NFCReaderX:v
After that please scan again, you should see in the log something like:
'' patchInfo = 0xdddddd"

Please upload this picture to this thread.

**Reply by parapenT1sta** [reply]
@ca00604 Can you help?

**Reply by jaky1977** [reply]
I'll see if someone helps me. thanks

**Reply by tzachi-dar** [reply]
still missing the line "patchinfo ="

**Reply by parapenT1sta** [reply]
@tzachi-dar I know that is why I deleted the previous message. I'm not using L2 at this moment, I'm asking a friend to send this info to me.

**Reply by parapenT1sta** [reply]
@tzachi-dar And how about now?
![Imagem WhatsApp 2023-05-24 às 12 15 44](https://github.com/NightscoutFoundation/xDrip/assets/608650/913d6992-defb-4d74-a6c6-6550bf59f042)

**Reply by jaky1977** [reply]
Other
![Screenshot_2023-05-24-13-27-00-442_com facebook orca-edit](https://github.com/NightscoutFoundation/xDrip/assets/56890091/fdd8af26-fc54-468c-92f6-ac5274a354e7)

**Reply by Cheun1** [reply]
![image](https://github.com/NightscoutFoundation/xDrip/assets/41595891/75a83d6a-e310-40e0-914b-40854a0515e4)

**Comment by tzachi-dar** [comment]
yes, this is what I was looking for.
Hope to have the new build tonight.

**Reply by parapenT1sta** [reply]
Thank you a lot!!! :-) Waiting for it.

**Reply by jaky1977** [reply]
thanks you

**Reply by juanpmartin** [reply]
Hi Tzachi! thanks a lot for your help and support. I'm in Argentina and Libre2 was just launched. I've check in the code and looks like the sensor info is not yet included. I'm reading 0x2b0a3a as you can see in the capture below. Would you be so kind in adding it to he switch statement? Once again, thanks a lot.

![Screenshot_20240810_234525_xDrip+](https://github.com/user-attachments/assets/a31cadf4-2e65-4c55-8792-81cf7f6870d7)

**Comment by parapenT1sta** [comment]
@tzachi-dar Can you check my suggestion here https://github.com/NightscoutFoundation/xDrip/discussions/2845#discussion-5229793?

**Comment by parapenT1sta** [comment]
The fix is available at https://github.com/NightscoutFoundation/xDrip/discussions/2845#discussioncomment-5995246
